docs/intro.rst

```diff
@@ -135,6 +164,14 @@ Keep in mind that CritiqueBrainz only supports importing into an empty database.
 This should work if you just ran ``init_db`` command.
 
 
+Running
+-------
+
+You can start the Webserver using the following command::
+
+  docker-compose -f docker/docker-compose.dev.yml run critiquebrainz python3 manage.py runserver
```

Comment:
Umm, shouldn't everything get started by some variant of `docker-compose up`. This is how it is done in LB and AB. @ferbncode, is this correct?

Reply:
I have no clue about the ```docker-compose up``` command. The command ```docker-compose -f docker/docker-compose.dev.yml run critiquebrainz python3 manage.py runserver``` which I wrote in the doc works, but tbh I don't know whether it is the correct way of running the server or not.

Reply:
`$ docker-compose -f docker/docker-compose.dev.yml up -d`

this command should work and is in the documentation.

Reply:
I think this block regarding how to start the webserver should be removed.